recipes/nginx.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Cookbook Name:: formsender
+# Recipe:: nginx
+#
+# Copyright 2015, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'osl-nginx'
```

Comment:
I have no preference, I am not as familiar with haproxy and how we use it here, but if it can talk wsgi, it sounds like a fine solution to me.